### DISCUSSION TITLE
Defining your environments when targeting multiple platforms ?

### DISCUSSION THREAD

**Discussion by rthery** [getsentry/sentry-unity#850] [Q&A] [answered]
Hello,

When releasing the same game on multiple platforms (so releasing to iOS and Android to give the most common scenario with Unity), what would be the suggested way to handle the different platforms ?

I don't like the idea of having one project per platform, because it will split issues.
Right now we have a `development` and a `production` environment, and while for issues this is fine because we can filter per `os.name`, all the project level views such as Releases will contained both Android and iOS, making it impossible to know one platform specific crash rate for example (without going on the platform own admin).

Now I came to the conclusion that maybe we need to split our environments per platforms, we will end up with `development-ios`, `development-android`, etc.

Does that sounds like the best approach ? Or is there a better way ?
Thanks for your feedback!

By the way if this is the way to go, I'll make a suggestion to edit https://docs.sentry.io/platforms/unity/configuration/environments/ to give some concrete example when supporting multiple Unity platforms.

**Comment by bitsandfoxes** [comment, marked as answer]
Hey @rthery,
so afaik your solution is a pretty good workaround to the current limitations of the Release health feature and I'm not aware of any other solution to that problem.
I'll try to update the docs to reflect that.

**Reply by rthery** [reply]
Thanks for your response! We'll try to go with this then :)
